x-pack/plugins/security/public/management/roles/roles_grid/roles_grid_page.tsx

```diff
@@ -183,7 +183,6 @@ export class RolesGridPage extends Component<Props, State> {
 
         <EuiInMemoryTable
           itemId="name"
-          responsiveBreakpoint={false}
```

Comment:
Without responsive breakpoints looks too dense and sort of broken. See screenshot in the PR description.

Reply:
> Without responsive breakpoints looks too dense and sort of broken. See screenshot in the PR description.
> 
> cc @legrego to confirm that such UI change would be okay.

I don't remember why we explicitly disabled this in the past, but I think allowing the responsive breakpoint makes a lot of sense for this screen. I'm happy for you to include this in the PR. Thanks!